### PR TITLE
Vendor tweaks & prices: take 2

### PR DESCRIPTION
**Pull request by Mickyan**
:cl: Mickyan
tweak: the order in which items are listed in certain vendors has been changed, giving priority to cheaper items and occasionally sorting by category
balance: prices for a number of items have been changed, certain vendors have new premium merchandise in stock
balance: decreased the amount of reagents contained in ramen cups & hot coco
fix: Chaplains no longer have to pay for their equipment
/:cl:

It's a lot of little changes, I'll try to list everything that's notable. 

Youtool: added utility belt, welding mask, earmuffs & industrial welder as premium items

Medical: adjusted prices for a number of items (original prices were kind of phoned in no offence, 25 credits for a pinpointer?). Basic supplies mostly untouched aside from a slight increase in price for medsprays compared to bottles. Synthflesh sprays available as premium items.

Dinnerware: knives no longer premium with a slight bump up in price. Cleavers are more expensive.

Clothesmate: ordinary items (ie. beanies, scarves, regular pants etc) are far less expensive, sorted on top by category, increased stock

Booze-o-Mat: added ethanol bottles as a premium item, reordered the list into categories with items that are bought more frequently generally being on top

Snack Machine: prices adjusted to scale with the amount of nutrition they give, based on the standard values of jerky/chips/candy. Ramen cups contain less reagents, it was so good compared to the rest I would have had to price it at 130 credits. Protip: based on the amount of nutrition given, ramen is still the cheapest option by virtue of needing an extra step to prepare

Coffee Machine: decresed stock to be more in line with food vendors, adjusted price and amount of reagents of hot coco to be more in line with snack vendor

Janivend: water tank and galoshes are more expensive (mostly just stops assistants from easily walking in and essentially stealing them)

### CODE REVIEW

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 1]
```diff
@@ -236,19 +236,20 @@
...
+	list_reagents = list("hot_coco" = 15, "sugar" = 5)
 	foodtype = SUGAR
-
 	resistance_flags = FREEZE_PROOF
+	custom_price = 42
 
 
 /obj/item/reagent_containers/food/drinks/dry_ramen
 	name = "cup ramen"
 	desc = "Just add 10ml of water, self heats! A taste that reminds you of your school years."
```
Comment (JStheguy) [inline code/modules/food_and_drinks/drinks/drinks.dm]: Absolutely anal, but this should be 5ml of water now that you've halved the ramen content.

Comment (Mickyan) [reply code/modules/food_and_drinks/drinks/drinks.dm]: yeah that's fair

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -351,6 +354,7 @@
 	item_state = "lb_suit"
 	item_color = "overalls"
 	can_adjust = FALSE
+	custom_price = 20
```
Comment (Iamgoofball) [inline code/modules/clothing/under/miscellaneous.dm]: why are you making cosmetics cheap

this is a bad idea

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 3]
```diff
@@ -236,19 +236,20 @@
...
+	custom_price = 42
 
 
 /obj/item/reagent_containers/food/drinks/dry_ramen
 	name = "cup ramen"
-	desc = "Just add 10ml of water, self heats! A taste that reminds you of your school years."
+	desc = "Just add 5ml of water, self heats! A taste that reminds you of your school years."
 	icon_state = "ramen"
-	list_reagents = list("dry_ramen" = 30)
+	list_reagents = list("dry_ramen" = 15)
```
Comment (Iamgoofball) [inline code/modules/food_and_drinks/drinks/drinks.dm]: uh

**Comment by ExcessiveUseOfCobblestone** [comment]
I know it's a new feature but do we really have to bundle a bunch of very different changes?

**Comment by Mickyan** [comment]
yeah, I'm not making a different PR for every item I change, would have been nice if prices had not been half assed in the first place but that's not what we got

And most changes relate to one another anyway, except the fix I guess

**Comment by epochayur** [comment]
>Youtool: added utility belt

this changes everything

**Comment by optimumtact** [comment]
>It's a lot of little changes, I'll try to list everything that's notable.
No, you'll list every fucking thing you changed

**Comment by adamkad1** [comment]
remove goof when

**Pull request merged**